android/app/buildConfigs.gradle

```diff
@@ -48,7 +58,13 @@ def setupGoogleServices(buildConfig, resValue) {
 }
 
 def setupResourceValues(buildConfigName, resValue) {
-    def command = "yarn --silent babel-node tools/build-config to-xcconfig $buildConfigName"
+    def command
+    def baseCommand = "yarn --silent babel-node tools/build-config to-xcconfig $buildConfigName"
```

Comment:
This code can be deduplicated by moving it to a separate function: executeProcess(command) which returns the output as string